src/js/disability-benefits/526EZ/components/ReviewCardField.jsx

```diff
@@ -181,14 +181,18 @@ export default class ReviewCardField extends React.Component {
 ReviewCardField.propTypes = {
   uiSchema: PropTypes.shape({
     'ui:options': PropTypes.shape({
-      viewComponent: PropTypes.element.isRequired
+      viewComponent: PropTypes.oneOfType(
+        [PropTypes.element, PropTypes.func]
+      ).isRequired
     }).isRequired
   }).isRequired,
   schema: PropTypes.object.isRequired,
   errorSchema: PropTypes.object.isRequired,
   idSchema: PropTypes.object.isRequired,
   registry: PropTypes.shape({
-    SchemaField: PropTypes.element.isRequired,
+    fields: PropTypes.shape({
```

Comment:
This was throwing a prop type warning because I forgot it was `registry.fields.SchemaField`.